website/projects/scimaps/src/app/pages/books/books.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="book" *ngFor="let book of books">
```

Comment:
No need for a wrapping div. Place ngFor and class directly on sci-book-overview

Reply:
I had that initially but for some reason my styles weren't working when the class was on the sci-book-overview element.  I'll look at it again

Reply:
Generally you should have
```
:host {
  display: block;
  ...
}
```
in all component scss to ensure it will work correctly